docs/gestor_tareas.md

```diff
@@ -0,0 +1,26 @@
+# Gestor de tareas
+En este caso al estar trabajando en Python, la herramienta de construcción que estamos usando como [gestor de dependencias](./gestor_dependencias.md) lo podemos usar para automatizar ciertas tareas, en este caso queremos automatizar la instalación de las dependencias, que es capaz de realizar apoyándose en el archivo [pyproject.toml](../pyproject.toml). 
+
+## Automatización de tareas
+Usar Makefile para ejecuar herramientas en Python es una práctica común, especialmente en proyectos en los que se requieren múltiples tareas repetitivas, como puedens ser pruebas. 
+
+Makefile facilita la automatización de estas tareas con comandos simples y mejorando la organización y eficiencia en el flujo de trabajo. Otras opciones para automatizar tareas disponibles en Python:
+### 1. Scripts de Python personalizados
+Se puedes escribir scripts en Python que que realice las tareas de configuración, pruebas, limpieza, instalación, etc. Ofrece gran flaxibilidad ya que se puede aprovechar la programación completa en Python, además funciona en cualquier entorno donde Python esté instalado.   
+Sin embargo, cada comando requiere llamadas a subprocess, lo que puede hacer que el script sea más detallado y extenso. También se suma que ejecutar tareas individuales requiere conocer la interfaz de Python y el nombre de cada función
+### 2. Invoke
+Es una biblioteca de Python para automatizar tareas que permite definir comandos y ejecutarlos desde la terminal, similar a make. Ofrece una sintaxis elegante y es multiplataforma, lo que lo hace atractivo para proyectos que necesitan ejecutarse en Windows.
```

Comment:
Una cosa interesante que debéis mirar es si se actualiza con mucha frecuencia. Podéis simplemente mirar al repositorio. ¿Has mirado el de invoke?

Reply:
¿Por qué lo marcas como resuelto?

Reply:
En este commit c1aa9c2f4f13bb5cc632a23e8968693d2d9c9cfd, añado información en la línea 14 sobre la última actualización de Invoke, pensaba que era suficiente